components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/dao/RegistryPolicyDAOImpl.java

```diff
@@ -84,6 +88,7 @@ public RegistryPolicyDAOImpl() {
     @Override
     public void init(Properties properties) {
 
+        // Do nothing
```

Comment:
Shall we avoid overriding this?

Reply:
Reintroduced the legacy implementation where we set the pdp path reading entitlement properties. Earlier impl seemed to disregard the value set in the properties file.